django/db/migrations/writer.py

```diff
@@ -184,6 +185,9 @@ def as_string(self):
         if self.migration.replaces:
             items['replaces_str'] = "\n    replaces = %s\n" % self.serialize(self.migration.replaces)[0]
 
+        if self.migration.initial:
```

Comment:
I'm setting
                         instance.initial = app_label not in self.existing_apps
in autodetector, so when this is True, we need to write out `initial = True`, the tri-value is only in play when
creating migration from a migration file.